### DISCUSSION TITLE
differentially expressed genes

### DISCUSSION THREAD

**Discussion by feanaros** [GreenleafLab/ArchR#1633] [Questions / Documentation]
Hi developers! Thank you for this wonderful package! a very basic question I have. Is it possible to find differentially expressed genes using  getmarkerfeatures?

 markersGS <- getMarkerFeatures(ArchRProj = archrproject,
                                useMatrix = "GeneIntegrationMatrix",
                                groupBy = "Clusters",
usegroups= "Red_KO",
bdgGroups= "Red_WT"
                                bias= c("TSSEnrichment", "log10(nFrags)"),
                                testMethod = "wilcoxon",
                                binarize = F, 
                                maxCells = 1000) 

 markerList_genes <- getMarkers(markersGS, cutOff = "FDR <= 0.05 & Log2FC >= 0.5") 
 The markerlist genes resulting contains empty rows and 9 column. Where I'm wrong? Thanks!!!!!
F

**Comment by rcorces** [comment]
Please note that this is not an Issue, but a Discussion and should have been posted there.
Presumably, the filters you have set via `cutOff` result in zero pass-filter features.

**Reply by feanaros** [reply]
Ok, with a less stringent cutoff, I found 0 features. I saw that FDR are higher than 0.5, so presumably the genes are not statistically significatives. However, with Seurat I found at least 20 DE genes setting the cutoff very stringent, so I cannot explain that. The integrationMatrix comes from the same scRNAseq that I used for Seurat.
